example/cycle/test/test.js

```diff
@@ -7,31 +7,25 @@ import * as actions from '../../actions';
 
 import { fetchReposByUser, searchUsers } from '../';
 
-function assertSourcesSinks(sources, sinks, main, done) {
-  const Time = mockTimeSource();
+function assertSourcesSinks(sources, sinks, main, done, timeOpts = {}) {
```

Comment:
Imho, not sure if it should go into cycle/time. I see cycle/time as a lower abstraction to also enable build utilities like that on top of it. But I definitely see this into a separate test-helper library for cyclejs. Curious to hear what @Widdershin think about this! (and keen to help making this happen)